aspnetcore/hosting/aspnet-core-module.md

```diff
@@ -13,17 +13,46 @@ uid: hosting/aspnet-core-module
 
 By [Luke Latham](https://github.com/GuardRex), [Rick Anderson](https://twitter.com/RickAndMSFT) and [Sourabh Shirhatti](https://twitter.com/sshirhatti)
 
-In ASP.NET Core, the web application is hosted by an external process outside of IIS. The ASP.NET Core Module is an IIS 7.5+ module, which is responsible for process management of ASP.NET Core http listeners and to proxy requests to processes that it manages. This document provides an overview of how to configure the ASP.NET Core Module for shared hosting of ASP.NET Core applications.
+ASP.NET Core Module is an IIS module that lets [Kestrel](../fundamentals/servers/kestrel.md) use IIS or IIS Express as a reverse proxy server. This document provides an overview of how to configure the ASP.NET Core Module for shared hosting of ASP.NET Core applications.
 
-## Installing the ASP.NET Core Module
+For an introduction to ASP.NET Core Module and information about how to install it, see [ASP.NET Core Module](../fundamentals/servers/aspnet-core-module.md).
 
-Install the [.NET Core Windows Server Hosting](https://go.microsoft.com/fwlink/?LinkId=827547) bundle on the server. The bundle will install the .NET Core Runtime, .NET Core Library, and the ASP.NET Core Module.
+## The Web.config file
 
-## Configuring the ASP.NET Core Module
+The ASP.NET Core Module is configured via a site or application *web.config* file and has its own `aspNetCore` configuration section within `system.webServer`. Here's an example from a Visual Studio project, with placeholders for settings that Visual Studio plugs in when you run or debug the project.
 
-The ASP.NET Core Module is configured via a site or application *web.config* file and has its own `aspNetCore` configuration section within `system.webServer`.
+````html
+<configuration>
+  <!--
+    Configure your application settings in appsettings.json. Learn more at http://go.microsoft.com/fwlink/?LinkId=786380
+  -->
+  <system.webServer>
+    <handlers>
+      <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified" />
+    </handlers>
+    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="true" stdoutLogFile=".\logs\stdout" forwardWindowsAuthToken="false" />
```

Comment:
Do these docs mention that stdout logging won't work if the "logs" directory (or whatever directory is configured for the  stdoutLogFile) doesn't already exist? This has tripped me up before.